src/extension.ts

```diff
@@ -26,7 +26,8 @@ export function activate(context: vscode.ExtensionContext): void {
     context.subscriptions.push(tomcatController);
     context.subscriptions.push(tomcatServerTree);
     context.subscriptions.push(vscode.window.registerTreeDataProvider('tomcatServerExplorer', tomcatServerTree));
-    context.subscriptions.push(vscode.commands.registerCommand('tomcat.tree.refresh', (element: TomcatServer) => tomcatServerTree.refresh(element)));
+    context.subscriptions.push(vscode.commands.registerCommand('tomcat.tree.refresh', (server: TomcatServer) => tomcatServerTree.refresh(server)));
```

Comment:
tomcatServerTree.refresh is enough for replaceing " (server: TomcatServer) => tomcatServerTree.refresh(server))"

Reply:
why... It does not work..